src/components/panel/panel.js

```diff
@@ -200,6 +201,34 @@ angular
  * @returns {!MdPanelAnimation} panelAnimation
  */
 
+/**
+ * @ngdoc method
+ * @name $mdPanel#newPanelGroup
+ * @description
+ * Creates a panel group and adds it to a tracked list of panel groups.
+ *
+ * @param groupName {string} Name of the group to create.
```

Comment:
The pattern that we use when declaring `@param` in all the other strings is `@param <type> <name>`, a few of the docstrings here use `@param <name> <type>`.